### DISCUSSION TITLE
Turkish sentences that should be supported

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#408] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Turkish** (`tr`) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by synesthesiam** [comment]
@alpdmrel I had to remove "çek" from the `HassOpenCover` intents for Turkish. With it, the same sentence like "Oturma Odasındaki Sol Perdeyi çek" could match either `HassOpenCover` or `HassCloseCover`

**Reply by alpdmrel** [reply]
Okay, You are right. I had missed that.

**Reply by alpdmrel** [reply]
Although is there going to be an option for HassToggleCover ?

**Reply by synesthesiam** [reply]
No, we decided not to have toggle intents for now.
